MmSupervisorPkg/Test/MmSupvRequestUnitTestApp/MmSupvRequestUnitTestApp.c

```diff
@@ -825,11 +827,18 @@ InspectSecurityPolicy (
   if (SvstLevel == UNDEFINED_LEVEL) {
     // Without specifying above, at most get level 20
     FinalLevel = MIN (FinalLevel, SMM_POLICY_LEVEL_20);
+    DEBUG ((DEBUG_INFO, "SvstLevel failure!  Returning level 20.\n"));
     goto Done;
   }
 
   FinalLevel = MIN (FinalLevel, SvstLevel);
 
+  DEBUG ((DEBUG_INFO, "No Level failure!  Printing all relevant levels\n"));

```

Comment:
Adding the final evaluation here is great, but I think we can make it even beefier by adding the prints into the individual verify functions. (i.e. what entry failed IO or MSR, as well as the memory attribute tests)

Reply:
Updated.

Reply:
Thanks for adding those! One more minor request: can we add some prints for IOMMU and TXT, please? :) The others looks legit.

Reply:
I added prints in those two functions.